app/components/date-time-modal/date-time-modal-controller.js

```diff
@@ -20,34 +34,47 @@ angular.module('bulbs.cms.dateTimeModal.controller', [
         // default to now if no time given
         $scope.tempDatetime = moment();
       }
-
-      $scope.TIMEZONE_LABEL = moment.tz(CmsConfig.getTimezoneName()).format('z');
+      $scope.tempTime = angular.copy($scope.tempDatetime);
 
       var timeNowWithOffset = function () {
         return moment.tz(CmsConfig.getTimezoneName());
       };
 
-      // callback function for using datetime calendar because it doesn't work at all in a sensible way
+      // callback function for using datetime calendar because it doesn't work
+      //  at all in a sensible way
```

Comment:
This comment should detail the non-sensible way in which you are working around, as is it offers no explanation.